scli-runner/src/main/scala/com.olegych.scastie.sclirunner/BspClient.scala

```diff
@@ -146,33 +147,34 @@ class BspClient(private val workingDir: Path,
         .map(r => r.getItems())
```

Comment:
add `.asScala` here so you can use it's api in further calls such as
`if (classes.size() == 0)` can become `if (classes.isEmpty)`. Try to always convert it to Scala API if you're doing anything more than getting the values.

You shouldn't also throw Errors here, instead try to return Optional value / Either[Erorr, Something].

If you decide to do so with Option or Either, you can even do `classes.headOption.flatMap(_.headOption)`

Reply:
Done hehe